docs/download.md

```diff
@@ -100,6 +100,10 @@ The list of all supported operations is provided [here](supported_ops.md).
 For a detailed list of changes, please refer to the
 [CHANGELOG](https://github.com/NVIDIA/spark-rapids/blob/main/CHANGELOG.md). 
 
+**_Note:_** Using the CUDA 450.80.02 driver in combination with the CUDA 10.1 or 10.2 toolkit may
```

Comment:
If this issue only happens on specific Driver version, could we ask users to use a different(or newer) version of driver other than 450.80.02?

Reply:
I don't think a driver with the fix is available yet.

Reply:
Updated the text to indicate a user can use a newer version of the driver.